cle/backends/relocations/peamd64.py

```diff
@@ -0,0 +1,21 @@
+import logging
+from . import generic_pe
+
+l = logging.getLogger('cle.relocations.peamd64')
+
+arch = 'peAMD64'
```

Comment:
> Since relocations are generally specific to a backend, perhaps each backend module should have its own relocations subdir

@rhelmot What do you think?